### PR TITLE
Backport gRPC fixes

### PR DESCRIPTION
**Pull request by K900**
This fixes a segfault when stopping Murmur with gRPC enabled, probably worth having on the stable branch

### CODE REVIEW

**Comment by Krzmbrzl** [comment]
@K900 I think it would be best to squash your commits into a single one. Also I'm curious: why did you use the `m` prefix over the `b` one for a boolean flag?

File: src/murmur/MurmurGRPCImpl.h [hunk 1]
```diff
@@ -36,6 +36,7 @@ namespace MurmurRPC {
 class MurmurRPCImpl : public QThread {
 		Q_OBJECT;
 		std::unique_ptr<grpc::Server> m_server;
+		volatile bool m_isRunning;
```
Comment (Krzmbrzl) [inline src/murmur/MurmurGRPCImpl.h]: I'm curious: why did you use the `m_` prefix over the `b_` one for a boolean flag?

**Comment by K900** [comment]
Those commits aren't mine, they're part of #3906, I'm simply suggesting that they be backported from master to 1.3.x, as they fix a fairly annoying crash on server shutdown.

**Comment by Krzmbrzl** [comment]
Ah okay then.

Well to me this seams like a reasonable thing to do (provided we also draft a new release). THoughts @davidebeatrici ?

**Comment by davidebeatrici** [comment]
The issue is that official 1.3 builds don't provide support for gRPC.

Backporting the fixes sounds good to me, but they would only be beneficial to people building from source.

**Comment by K900** [comment]
The Arch packages definitely build with gRPC enabled, which is my primary motivation in this.

**Pull request merged**